BusyList/TaskItem.cs

```diff
@@ -4,24 +4,31 @@ namespace BusyList
 {
     public class TaskItem
     {
-        public TaskItem(int id, string description, TaskStatus status = TaskStatus.NotStarted)
+        public TaskItem(int id, string description, string priority, TaskStatus status = TaskStatus.NotStarted)
```

Comment:
Is `string` the best data type to store priority in? What alternatives can you think of?